workflows/prognostic_c48_run/runtime/logs.py

```diff
@@ -83,6 +85,32 @@ def capture_fv3gfs_funcs():
         setattr(wrapper, func, captured_stream(getattr(wrapper, func)))
 
 
+def setup_file_logger(name):
+    """Configure a logger which streams to name.txt as well as stderr."""
+    logger = logging.getLogger(name)
+    fh = logging.FileHandler(f"{name}.txt")
+    fh.setLevel(logging.INFO)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.INFO)
+    ch.setFormatter(logging.Formatter(fmt="%(levelname)s:%(name)s:%(message)s"))
+
+    logger.addHandler(fh)
+    logger.addHandler(ch)
+
+
+def log_mapping(time, content, logger_name):
```

Comment:
I'd add some typing information to the function since this is the main contact point with the runfile.

`time: "datetime-like"`??, `content: Mapping[str, Any]` (I don't think here's a way to specify "serializable"), `logger_name: str`

Reply:
Good idea.